pipeline/dist/dist.go

```diff
@@ -52,3 +56,14 @@ func mkdir(ctx *context.Context) error {
 	// #nosec
 	return os.MkdirAll(ctx.Config.Dist, 0755)
 }
+
+func isSneaky(ctx *context.Context) error {
+	dir, err := filepath.Abs(ctx.Config.Dist)
+	if err != nil {
+		return err
+	}
+	if dir == "" || dir == "/" {
```

Comment:
thinking more about it now, ideally the OS won't let us delete the `/` anyway, unless goreleaser was run by `root`.

is that the case?

Reply:
That should be correct: `RemoveAll removes path and any children it contains. It removes everything it can but returns the first error it encounters.` [os.RemoveAll](https://golang.org/pkg/os/#RemoveAll). Shall we expect goreleaser to never run as root?

Reply:
I think it is a safe assumption...